release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3929,3 +3929,29 @@ $ oc adm release info 4.13.35 --pullspecs
 [id="ocp-4-13-35-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-36"]
+=== RHSA-2024:1037 - {product-title} 4.13.36 bug fix and security update
+
+Issued: 2024-03-06
+
+{product-title} release 4.13.36, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:1037[RHSA-2024:1037] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:1039[RHBA-2024:1039] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.36 --pullspecs
+----
+
+[id="ocp-4-13-36-bug-fixes"]
+==== Bug fixes
+
+* Previously, a race condition between udev events and the creation queues associated with physical device led to some of the queues being configured with the wrong Receive Packet Steering (RPS) mask while they should be reset to zero. This resulted in the RPS mask being configured on the queues of the physical devices, meaning they were using RPS instead of Receive Side Scaling (RSS) which could impact the performance. With this fix, the event was changed to be triggered per queue creation instead of on device creation. This guarantees that no queue will be missing. The queues of all physical devices are now set up with the correct RPS mask which is empty.
```

Comment:
```suggestion
* Previously, a race condition between `udev` events and the creation queues associated with physical device led to some of the queues being configured with the wrong Receive Packet Steering (RPS) mask while they should be reset to zero. This resulted in the RPS mask being configured on the queues of the physical devices, meaning they were using RPS instead of Receive Side Scaling (RSS) which could impact the performance. With this fix, the event was changed to be triggered per queue creation instead of on device creation. This guarantees that no queue will be missing. The queues of all physical devices are now set up with the correct RPS mask which is empty.
```

Reply:
Match [4.14](https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-4:~:text=Currently%2C%20due%20to%20a%20race%20condition%20between%20the%20application%20of%20a%20udev%20rule%20on%20physical) `udev` reference.